UnityGLTF/Assets/UnityGLTF/Scripts/Loader/WebRequestLoader.cs

```diff
@@ -17,11 +17,14 @@ public class WebRequestLoader : ILoader
 	{
 		public Stream LoadedStream { get; private set; }
 
+		public bool HasSyncLoadMethod { get; private set; }
```

Comment:
Can be HasSyncLoadMethod => false;

Reply:
I noticed you have started CI. Are you OK with leaving it as it is?

Reply:
Yeah it's not supported in this version of .NET

Reply:
Got it!